src/functions-reference/compound_arithmetic_and_assignment.Rmd

```diff
@@ -29,274 +11,111 @@ cat(' * <a href="compound-elementwise-division-and-assignment.html">Compound Ele
-\index{{\tt \bfseries operator\_compound\_multiply }!{\tt (real x, real y): void}|hyperpage}
-
-`void` **`operator*=`**`(real x, real y)`<br>\newline
-`x *= y` is equivalent to `x = x * y`.
-`r since("2.17")`
-
-<!-- void; operator*=; (vector x, real y); -->
-\index{{\tt \bfseries operator\_compound\_multiply }!{\tt (vector x, real y): void}|hyperpage}
-
-`void` **`operator*=`**`(vector x, real y)`<br>\newline
-`x *= y` is equivalent to `x = x * y`.
-`r since("2.17")`
-
-<!-- void; operator*=; (row_vector x, real y); -->
-\index{{\tt \bfseries operator\_compound\_multiply }!{\tt (row\_vector x, real y): void}|hyperpage}
-
-`void` **`operator*=`**`(row_vector x, real y)`<br>\newline
-`x *= y` is equivalent to `x = x * y`.
-`r since("2.17")`
-
-<!-- void; operator*=; (matrix x, real y); -->
-\index{{\tt \bfseries operator\_compound\_multiply }!{\tt (matrix x, real y): void}|hyperpage}
+Compound multiplication and assignment works wherever the corresponding
+multiplication and assignment would be well formed.
 
-`void` **`operator*=`**`(matrix x, real y)`<br>\newline
-`x *= y` is equivalent to `x = x * y`.
-`r since("2.17")`
+<!-- void; operator*=; (T x, U y); -->
+\index{{\tt \bfseries operator\_compound\_mulitply }!{\tt (T x, U y): void}|hyperpage}
 
-<!-- void; operator*=; (row_vector x, matrix y); -->
-\index{{\tt \bfseries operator\_compound\_multiply }!{\tt (row\_vector x, matrix y): void}|hyperpage}
+`void` **`operator*=`**`(T x, U y)`<br>\newline
+`x *= y` is equivalent to `x = x * y`.  Defined for all types `T` and `U`
+where `T = T * U` is well formed.
+`r since("2.17, complex signatures added in 2.30")`
 
-`void` **`operator*=`**`(row_vector x, matrix y)`<br>\newline
-`x *= y` is equivalent to `x = x * y`.
-`r since("2.17")`
 
-<!-- void; operator*=; (matrix x, matrix y); -->
-\index{{\tt \bfseries operator\_compound\_multiply }!{\tt (matrix x, matrix y): void}|hyperpage}
-
-`void` **`operator*=`**`(matrix x, matrix y)`<br>\newline
-`x *= y` is equivalent to `x = x * y`.
-`r since("2.17")`
 
 ## Compound division and assignment
 
-<!-- void; operator/=; (int x, int y); -->
-\index{{\tt \bfseries operator\_compound\_divide }!{\tt (int x, int y): void}|hyperpage}
-
-`void` **`operator/=`**`(int x, int y)`<br>\newline
-`x /= y` is equivalent to `x = x / y`.
-`r since("2.17")`
-
-<!-- void; operator/=; (real x, real y); -->
-\index{{\tt \bfseries operator\_compound\_divide }!{\tt (real x, real y): void}|hyperpage}
-
-`void` **`operator/=`**`(real x, real y)`<br>\newline
-`x /= y` is equivalent to `x = x / y`.
-`r since("2.17")`
+Compound division and assignment works wherever the corresponding
+division and assignment would be well formed.
 
-<!-- void; operator/=; (vector x, real y); -->
-\index{{\tt \bfseries operator\_compound\_divide }!{\tt (vector x, real y): void}|hyperpage}
+<!-- void; operator/=; (T x, U y); -->
+\index{{\tt \bfseries operator\_compound\_mulitply }!{\tt (T x, U y): void}|hyperpage}
```

Comment:
```suggestion
\index{{\tt \bfseries operator\_compound\_divide }!{\tt (T x, U y): void}|hyperpage}
```

Reply:
Thanks for catching all these typos.  Fixed this one, too.

Reply:
Hmmm.  It's making me comment again before letting me resolve this conversation.